docs/WifiPerformance.md

```
@@ -1202,128 +571,129 @@ This section presents the performance test results, including key metrics and te
 
     ```
 
-#### Realtek 8723BU
+#### UWE 5622
 
-<img src=https://stuff.armbian.com/netbox/media/devicetype-images/RTL8723BU.png>
-<span style="font-size: 0.5rem;">OS: Armbian v25.5.0-trunk.550, 6.12.28-current-x86</span>
+<img src=https://stuff.armbian.com/netbox/media/devicetype-images/UWE5622.png>
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)